.stack-8.0.2.yaml

```diff
@@ -0,0 +1,5 @@
+resolver: lts-9.1
+packages:
+- '.'
+- protobuf/
+- protobuf-lens/
```

Comment:
Could you either merge this file with `stack.yaml` making the default GHC 8.0.2 (you'd need to change the `.travis.yaml` file back) that way a fresh checkout would just work with `stack build`.